udata/templates/home.html

```diff
@@ -57,7 +57,7 @@ <h1 class="page-header">
 
                 <div class="row">
                     <ul class="card-list">
-                        {% for dataset in current_site.settings.home_datasets %}
+                        {% for dataset in current_site.settings.home_datasets|rejectattr('deleted') %}
```

Comment:
Because of the stock, new deletion willl trigger the pull, but this avoid failure on existing ones.